iris-mpc-cpu/src/execution/hawk_main.rs

```diff
@@ -515,6 +515,7 @@ impl JobSubmissionHandle for HawkHandle {
                 successful_reauths: vec![],                             // TODO.
                 reauth_target_indices: Default::default(),              // TODO.
                 reauth_or_rule_used: Default::default(),                // TODO.
+                modifications: HashMap::new(),                          // TODO.
```

Comment:
Same as in the GPU actor:

```suggestion
                modifications: batch.modifications,
```